cpp/open3d/core/kernel/UnaryEWCPU.cpp

```diff
@@ -130,6 +130,19 @@ void CopyCPU(const Tensor& src, Tensor& dst) {
         MemoryManager::Memcpy(dst.GetDataPtr(), dst.GetDevice(),
                               src.GetDataPtr(), src.GetDevice(),
                               src_dtype.ByteSize() * shape.NumElements());
+    } else if (dst.IsContiguous() && src.NumElements() == 1) {
+        DISPATCH_DTYPE_TO_TEMPLATE_WITH_BOOL(src_dtype, [&]() {
+            using src_t = scalar_t;
+            DISPATCH_DTYPE_TO_TEMPLATE_WITH_BOOL(dst_dtype, [&]() {
+                using dst_t = scalar_t;
+                dst_t value = static_cast<dst_t>(src.ToFlatVector<src_t>()[0]);
```

Comment:
use `src.Item<src_t>` to get the source value.

Reply:
Done. Still, it didn't resolve the issue for `FillObject`, where dtype is initialised as `core::Dtype dtype(core::Dtype::DtypeCode::Object, byte_size, class_name);`  [Link to test](https://github.com/intel-isl/Open3D/blob/692dc22ff611388d57d3c3a7c8c040fdee4eb232/cpp/tests/core/TensorObject.cpp#L114).

Can we support this case of we need to find out a way to avoid this case, such as if there is a way to check Dtype is for objects or iterate of other standard datatypes in if statement condition?

Reply:
The simple fix would be:
`} else if (dst.IsContiguous() && src.NumElements() == 1 && !src_dtype.IsObject()) {`